requirements.txt

```diff
@@ -12,3 +12,5 @@ Jinja2
 netaddr
 
 pbr!=2.1.0,>=2.0.0 # Apache-2.0
+
+looseversion;python_version >= '3.12'
```

Comment:
how does this work when being used in a classic charm?

Reply:
Based on your comment above, it probably breaks unless the parent charm ensures that the package is installed.